src/domain/application/components/ApplicationContacts.tsx

```diff
@@ -36,6 +37,37 @@ function getEmptyCustomerWithContacts(): CustomerWithContacts {
   };
 }
 
+function isRegistryKeyInputEnabled(
+  customerType: string,
+  selectedContactType: string | null,
```

Comment:
`keyof typeof ContactType | null` could be used here. And same in getRegistryKeyLabel function.

Reply:
Thank you for the correction! Fixed this.